test/code_size/hello_wasm_worker_wasm.json

```diff
@@ -3,8 +3,8 @@
   "a.html.gz": 433,
   "a.js": 693,
   "a.js.gz": 464,
-  "a.wasm": 1862,
-  "a.wasm.gz": 1040,
-  "total": 3292,
-  "total_gz": 1937
+  "a.wasm": 1896,
+  "a.wasm.gz": 1060,
+  "total": 3326,
+  "total_gz": 1957
```

Comment:
Given that `__builtin_wasm_tls_align()` is a compile-time constant, I hope with help of Binaryen's optimizer we'd be able to do this fix without increasing code size for this test.

Reply:
Looks like it worked..  in fact we got a tiny code size improvment!